deltachat-jsonrpc/src/api/types/message.rs

```diff
@@ -85,6 +85,8 @@ pub struct MessageObject {
 
     webxdc_info: Option<WebxdcMessageInfo>,
 
+    webxdc_href: Option<String>,
```

Comment:
```suggestion
    /// On a WebxdcInfoMessage this might include a hash holding
    /// information about a specific position or state in a webxdc app
    webxdc_href: Option<String>,
```